pom.xml

```diff
@@ -71,17 +72,18 @@
                 </exclusion>
             </exclusions>
         </dependency>
-        <dependency>
-            <groupId>org.springframework.security</groupId>
-            <artifactId>spring-security-test</artifactId>
-            <scope>test</scope>
-        </dependency>
```

Comment:
re-arranged code -> section moved up to where dependecy for spring-boot-starter-security is